docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:bionic
+
+COPY vm-discovery /usr/local/bin/
+USER 1337:1337
```

Comment:
is this user important for any reason?  if so, should we document?

Reply:
I think without it it should work. I copy & pasted this line from pilot discovery Dockerfile and I think this user is important if it has proxy. But now this sidecar only interacts with kube, so that should be fine. I will remove this line for now.